AdminUi/apps/admin_ui/lib/core/widgets/filters/number_filter.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:admin_api_sdk/admin_api_sdk.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import '/core/constants.dart';
+
+class NumberFilter extends StatefulWidget {
+  final void Function(FilterOperator operator, String enteredValue) onNumberSelected;
+  final String label;
+
+  const NumberFilter({
+    required this.onNumberSelected,
+    required this.label,
+    super.key,
+  });
+
+  @override
+  State<NumberFilter> createState() => _NumberFilterState();
+}
+
+class _NumberFilterState extends State<NumberFilter> {
+  late FilterOperator _operator = FilterOperator.equal;
+  late String _value = '';
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          '${widget.label}:',
+          style: const TextStyle(fontWeight: FontWeight.bold),
+        ),
+        Gaps.h8,
+        Row(
+          children: [
+            DropdownButton<FilterOperator>(
+              value: _operator,
+              onChanged: (newValue) {
+                setState(() => _operator = newValue!);
+                widget.onNumberSelected(_operator, _value);
```

Comment:
you should access newValue instead of _value here

Reply:
Hmm.. maybe not, because _value is the value of a input and newValue in here is the value of an operator.